docs/Displaying a User Interface/Media Clock/index.md

```diff
@@ -228,7 +231,95 @@ sdlManager.sendRpc(mediaClock);
 The audio indicator is, essentially, the play / pause button. You can tell the system which icon to display on the play / pause button to correspond with how your app works. For example, if audio is currently playing you can update the play/pause button to show the pause icon. On older head units, the audio indicator shows an icon with both the play and pause indicators and the icon can not be updated.
 
 For example, a radio app will probably want two button states: play and stop. A music app, in contrast, will probably want a play and pause button. If you don't send any audio indicator information, a play / pause button will be displayed.
+
+## Setting The Media Forward / Back Button Style (RPC v7.1+)
+As of RPC v7.1, you can set the style of the media forward / back buttons to show icons for skipping time (in seconds) forward and backward instead of skipping tracks. The skipping time style is common in podcast & audiobook media apps.
+
+When you set the skip indicator style, you can set type `TRACK`, which is the default style that shows "skip forward" and "skip back" indicators. This is the only style available on RPC < 7.1 connections. You can also set the new type `TIME`, which will allow you to set the number of seconds and display indicators for skipping forward and backward in time.
+
+### Track Style
+![Generic - Seek Indicator Type TRACK](assets/generic_mediaclock.png)
+
+@![iOS]
+##### Objective-C
+```objc
+SDLSeekStreamingIndicator *trackStyle = [[SDLSeekStreamingIndicator alloc] initWithType:SDLSeekIndicatorTypeTrack];
+SDLSetMediaClockTimer *mediaClock = [SDLSetMediaClockTimer countUpFromStartTimeInterval:0 toEndTimeInterval:300 playPauseIndicator:SDLAudioStreamingIndicatorPause forwardSeekIndicator:trackStyle backSeekIndicator:trackStyle countRate:nil];
+[self.sdlManager sendRequest:mediaClock];
+```
+
+##### Swift
+```swift
+let trackStyle = SDLSeekStreamingIndicator(type: .track)
+let mediaClock = SDLSetMediaClockTimer.countUp(from: 0, to: 300, playPauseIndicator: .pause, forwardSeekIndicator: trackStyle, backSeekIndicator: trackStyle, countRate: nil)
+sdlManager.send(mediaClock)
+```
+!@
+
+@![android, javaSE, javaEE]
+```java
+// TODO: Add code example that sets backSeekIndicator and forwardSeekIndicator in for type TRACK
+```
+!@
+
+@![javascript]
+```js
+const streamingIndicator = new SDL.rpc.structs.SeekStreamingIndicator()
+    .setType(SDL.rpc.enums.SeekIndicatorType.TRACK);
+
+const mediaClock = new SDL.rpc.messages.SetMediaClockTimer()
+    .setUpdateMode(SDL.rpc.enums.UpdateMode.PAUSE)
+    .setForwardSeekIndicator(streamingIndicator)
+    .setBackSeekIndicator(streamingIndicator)
+    .setAudioStreamingIndicator(SDL.rpc.enums.AudioStreamingIndicator.PLAY);
+// sdl_javascript_suite v1.1+
+sdlManager.sendRpcResolve(mediaClock);
+// Pre sdl_javascript_suite v1.1
+sdlManager.sendRpc(mediaClock);
+```
+!@
+
+### Time Style
+![Generic - Seek Indicator Type TIME](assets/generic_sdlSeekStreamingIndicatorType_Time.png)
```

Comment:
Should this image be updated to match the code example?
i.e. forward skip indicator having 45

Reply:
Yeah, I agree